test/test_restful.py

```diff
@@ -100,6 +100,7 @@ def cmp(a, b):
 reg_process = None
 agent_process = None
 tenant_templ = None
+RUST_TEST = not bool(os.getenv("RUST_TEST"))
```

Comment:
Why is this `not`? If you're negating the env var value shouldn't this var be called something like `SKIP_RUST_TEST`?